tests/model_integration/test_model.py

```diff
@@ -21,6 +21,13 @@ def my_function(lm):
     assert str(lm) in ["this is a test another item1", "this is a test another item2"]
 
 
+def test_with_multitokenchars(selected_model: guidance.models.Model):
+    # Taken from https://github.com/guidance-ai/guidance/issues/934
+    lm = selected_model
+    lm += "歪" + select(["打正着", "门邪道"])
```

Comment:
For the record:
```python
>>> a
'打'
>>> b
'门'
>>> a.encode()
b'\xe6\x89\x93'
>>> b.encode()
b'\xe9\x97\xa8'
```
So the two leading characters do not share any common bytes here.